### PR TITLE
Add docker-compose with postgres

### PR DESCRIPTION
**Pull request by SocFoot**
https://trello.com/c/ZkhLTiIl/21-updating-a-template-code

We add docker-compose to the repo for quicker bootstrap.
This is meant for local use only and not for production use.
We focus on postgres but other dependencies can be added(redis).

### CODE REVIEW

File: templates/docker_compose.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+postgres_version = get_tools_version_of("postgres")
```
Comment (Nitemaeric) [inline templates/docker_compose.rb]: Do we have access to the ENV var set in `template.rb` at line 199?

```ruby
ENV["ASDF_POSTGRES_VERSION"] = get_tools_version_of("postgres")
```

I think we might be able to reuse that variable here.

Comment (SocFoot) [reply templates/docker_compose.rb]: done

Comment (Nitemaeric) [reply templates/docker_compose.rb]: I think you'll need to remove this line too.

**Review by Nitemaeric — APPROVED** [review]

**Review by peteryates — APPROVED** [review]
Looks good to me.

**Pull request merged**